src/main/java/frc/robot/util/Node.java

```diff
@@ -39,9 +42,9 @@ public Node()  {
    * @param row
    *  row it's in (1 = bottom, 2 = middle, 3 = top)
    * @param column
-   *  column from field boundary to other boundary (1 to 9)
+   *  column from field boundary to loading zone (1 to 9)
    */
-  public Node(Alliance alliance, int row, int column) {
+  public Node(RollerIntake intake, Alliance alliance, int row, int column) {
```

Comment:
Personaly I think the node class should be held in the node class. If the off set is going to be handled int he node class I think the game pice offset should be a method parameter for geting the pose.